cmd/backup/backup.go

```diff
@@ -34,7 +34,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type command config.CLIOptions
+type command struct {
+	config.CLIOptions
+}
```

Comment:
these need to be done because `type command config.CLIOptions` is the kind of type alias that doesn't copy functions, just fields.